ProteoformSuiteInternal/Envelope.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProteoformSuiteInternal
+{
+    public class Envelope
+    {
+        public int scan_num;
```

Comment:
Could you make these properties instead of fields?
`public int ScanNum { get; set; }` or `public int ScanNum { get; private set; }` if the property is only set within the constructor.